File: 1-two-sum/brute-force/brute-force.cs
public class Solution
{
    public int[] TwoSum(int[] nums, int target)
    {
        for (int i = 0; i < nums.Length; i++)
        {
            for (int j = 0; j < nums.Length; j++)
            {
                int sum = nums[i] + nums[j];

                if ((sum == target) && (i != j))
                {
                    return new int[] { i, j};
                }
            }
        }

        return new int[0];
    }
}

File: 1-two-sum/brute-force/brute-force.rs
impl Solution
{
    pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32>
    {
        for i in 0..nums.len()
        {
            for j in 0..nums.len()
            {
                let sum:i32 = nums[i] + nums[j];

                if ((sum == target) && (i != j))
                {
                    return vec![i as i32, j as i32];
                }
            }
        } 

        // No valid solution found
        vec![]
    }
}